modules/gathering-s2i-diagnostic-data.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-s2i.adoc
+
+[id="gathering-s2i-diagnostic-data_{context}"]
+= Gathering Source-to-Image diagnostic data
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. The S2I tool runs a build pod and a deployment pod in sequence, finally handing off to application pods once the deployment has completed successfully. Watch pod status throughout the S2I process to determine at which stage a failure occurs:
```

Comment:
Thank you Bob, I will keep this in mind in my writing.

Reply:
I have now restructured that paragraph so that it leads with the procedural instruction. I have also made it clearer as to the order in which the S2I process progresses:

~~~
. You can watch the Pod status throughout the S2I process to determine at which stage a failure occurs. The S2I tool runs a build Pod and a deployment Pod in sequence. The deployment Pod is responsible for deploying the application Pods based on the application container image created in the build stage.
~~~